cmd/kind/load/docker-image/docker-image.go

```diff
@@ -71,11 +71,15 @@ func NewCommand() *cobra.Command {
 
 func runE(flags *flagpole, cmd *cobra.Command, args []string) error {
 	imageName := args[0]
-	// Check that the image exists locally, if not return error
-	_, err := docker.ImageInspect(imageName)
+	// Check that the image exists locally and gets its ID, if not return error
+	lines, err := docker.ImageInspect(imageName, "{{ .Id }}")
```

Comment:
we should really break this out into a higher level abstraction, random packages should not be passing docker format strings and then parsing the output.

can we have `docker.ImageId(imageName)` instead?

Reply:
sure

Reply:
Thanks, it's not a huge deal now, but we will have to clean all of this up later, and I'd like to stop repeating this pattern as it was a mistake :^)